lib-vau-csharp/util/MediaTypeHeader.cs

```diff
@@ -1,6 +1,5 @@
-
 /*
- * Copyright 2025 gematik GmbH
+ * Copyright 2024 gematik GmbH

```

Comment:
```suggestion
 * Copyright 2025 gematik GmbH
```

Reply:
Yeah, I figured the headers will be recreated automatically upon a new release (if this gets merged).